power_grid_model_io_native_c/power_grid_model_io_native/include/power_grid_model_io_native/vnf_converter/vnf_pgm_converter.hpp

```diff
@@ -69,11 +84,24 @@ PgmVnfConverter::convert_input(power_grid_model::ConstDataset const* /*dataset*/
     convert_sym_gens_input();
     convert_links_input();
 
-    // then return the buffer
-    // return pgm_input_data;
-    // for now.
-    power_grid_model::ConstDataset* fake_data = nullptr;
-    return fake_data;
+    power_grid_model::Container<power_grid_model::Node> container;
+
+    for (const auto& node : nodes) {
+        container.emplace<power_grid_model::Node>(node.id(), node);
+    }
+
+    power_grid_model::meta_data::MetaData const meta_data;
```

Comment:
please use the meta_data provided by the power grid model from `meta_data_gen.hpp` (in https://github.com/PowerGridModel/power-grid-model/blob/feature/re-route-benchmark-validation-tests/power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/meta_data_gen.hpp#L26-L30 )